### PR TITLE
Document Makefile

### PR DESCRIPTION
**Pull request by pkarman**

### CODE REVIEW

File: Makefile.example [hunk 1]
```diff
@@ -7,72 +7,66 @@
...
-migrate:
+migrate:	## Run pending migrations
 	bundle exec rake db:migrate
 
-rollback:
+rollback:	## Rollback last migration
 	bundle exec rake db:rollback
 
-fresh:
+fresh:	## git master
```
Comment (lomky) [inline Makefile.example]: I don't follow this one?

File: Makefile.example [hunk 2]
```diff
@@ -7,72 +7,66 @@
...
 
-unsafe:
-	mv .git/hooks/pre-commit .git/hooks/pre-commit-linter
-
-safe:
-	mv .git/hooks/pre-commit-linter .git/hooks/pre-commit
-
-karma:
+karma:	## Start client karma server
 	cd client && node_modules/.bin/karma start
```
Comment (lomky) [inline Makefile.example]: Consider adding the `help` command line the Caseflow one has?
https://github.com/department-of-veterans-affairs/caseflow/blob/master/Makefile.example#L152

File: Makefile.example (hunk 1 above)
Comment (pkarman) [reply Makefile.example]: zapped it -- I don't actually use it

**Comment by pkarman** [comment]
thanks @lomky -- added help

**Pull request merged**